### PR TITLE
V2

### PR DESCRIPTION
**Pull request by darrynten**
Version 2 of Quick Dash

A major refactor with significant changes across the board.

The beginning of v2.

The purpose is to be able to include the package `@unicorn/quick-dash`
and begin a new dashboard project quickly and easily.

There is another project showing the boilerplate code to load the
project and configure.

### CODE REVIEW

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 8409e2b5e90eb8c7e32cf695238f1618b4a4152b into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-c940b994df15d2b800112783f0cb493c4f9ce7e5)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: index.html [hunk 1]
```diff
@@ -33,18 +33,12 @@
              xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid"
              style="margin: 0 auto; background: none;">
           <g transform="translate(50,50)">
-            <circle cx="0" cy="0" r="12.5" fill="none" stroke="#2962ff" stroke-width="4"
+            <circle cx="0" cy="0" r="12.5" fill="none" stroke="#9C27B0" stroke-width="4"
                     stroke-dasharray="39.269908169872416 39.269908169872416" transform="rotate(227.921)">
               <animateTransform attributeName="transform" type="rotate" values="0 0 0;360 0 0" times="0;1"
```
Comment (igorsergiichuk) [inline index.html]: Should we replace the `animateTransform` element with css animation?

File: src/main.js [hunk 2]
```diff
@@ -1,125 +1,56 @@
...
+
+comms.init()
+auth.init()
+lib.load()
+// lib.init()
+
+export default {
+  install: (Vue, options) => {
+    // router = [...options.router]
+    console.log(options.router, Vue.router, router)
```
Comment (igorsergiichuk) [inline src/main.js]: Is this line left for testing purposes?

Comment (darrynten) [reply src/main.js]: Yea that bit is WIP

File: index.html (hunk 1 above)
Comment (darrynten) [reply index.html]: Yes the svg animations are deprecated

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 833d2b16ca924d18ae1ae06aa0a4003eb4b18801 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-7be7bf18b94811e824a6c4ffb7aad29b55d58255)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 8262b2573afe683894b41ec7f464280b28ee6120 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-0b078610a8d631c055d98eb410dacb456016c800)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 305e7678e037694ea02f487c426c0f040c283ea0 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-31c2c3e12dd0ade754ef1b0f004a47841605eb9c)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 18ac6966ba73f636127954f7bd1207bed5486aa3 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-3a94c1e54947e1ecd3f57331a64f109c9ead6e22)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging d3f272c28cc707ada8b907bcba0bf9fdffc970b1 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-ae601afb1ab46d3720eb5a9e5271dfab6e92dce6)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 7b778ec1ad83234d037ea82f2223d61c3ffbf432 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-9ff7497b4f10510dcc2f8bc00e8688ee325f3909)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 25fd10ac60d9e56d1a2388028bc1e4877bf7625d into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-88b3d3399215b686c25f815c3a588aee21a466af)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 78678deec75d9ca6d02e69a74c0772aac3bc2102 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-f2f99688c35cdd57b9366aa6ca1e143ba3470c4f)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 67eeef908fbae35fcc40f403ab4ba053682dd6db into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-d4d57be3df4dfb3179ce022688f456dc462f81f7)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 5732585d8b554781d41011c60141f94c988a0a91 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-9d055c032fc3f4cb42b4b837250f2f2bb3b87a7b)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging d031539e1e39de64a80a470743093770daef7b98 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-9e5bcb8eb95352304db111c892831e1ee10e8900)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 6b758715c8aa9917eae88801b562272d1bf74682 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-e5f0bab116bca12a6517468937b0965d97d17261)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging c6eadf3ca1ea32d8f0f6fc389eebcdd14d0d3a70 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-9cf14977a78fe1a111d3087a85036af97fe94ba0)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging e123c276425cd3c204d103d1ebeee2bc8d5fcec0 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-b0e353724178616e2c3cf3ca909bc5cc5c6dcd6d)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging bd717bcd8fd4a47566ece673c3036c9f6fb72884 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-9f4e0b3f7318946510b3a8342ad818573541148d)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging e7238ed6c31d5f99057f50b13de177bc07eedb02 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-454b2aefd9fe524e598eaa8f4aa34f248c01eaad)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging f297f85279a8a72328a8c32bf2ac8ad476b39f24 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-0c14c35760013406b17949c3283926592b7fbe53)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 6bbe861c615bf16547580ff87cb184e48dbf44e3 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-16f15ea0836828db46adc6409d06eca7fdc2f5c3)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging c8889fe170d9ef373b825db2d1663e209dd5b119 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-a8f435ee0082755f72e8aa2f5a76bee1fe3cda4a)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging eb64fa24ed46b8f633cd2fcd2c061304f36459b6 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-f08428134360b598c37bbe1a01efef4f67bb2f65)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 1bfbe1a7d89d184c9e7286b6c0446f6fef675f2c into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-6a72cbc421a0f2b1f3dc216f22832bd94faeaab4)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 98e4f0daae287ba95bc496ccdd591b244702f330 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-ec86b841fe65b2be5bfc6737391a0fb51b1c7eb1)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 53022d253e8a7470918390e2e4a94c4f1ddd6d63 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-8e35e219c8fac80984c90edc1233e530940989c0)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging c06ce8c61715ddaeee187b7512f9a36bbadcf9e8 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-028756a57c9dc19b7123891f6a49252e4a773e70)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging c29bd5dd6d5236208aaa00e8cfd7c5813e2f2b52 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-7b1a2f8bc3ed10e9db6903d5465267e242553b4f)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 84cf2ea3e97060d06c403c936290e8b133ca511e into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-4f702a8d4e5bf2664b72d1ca13876c4d85cdd0f5)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging df8ba57de33b15dcff42ce10b857f9450e4f3b26 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-3e18de80532aa9190610eaf485e631e620be78dc)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging bf193ef5a07e86f4435c3ed1ff19f6508467bc1e into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-5d72f20ac828893b5dc3f158b1bcaf0c278b7625)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 33754b519b86ee362b80b3aea4d792e3ccd3cf94 into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-97062e8b7819b2933f6a9c2b04d109bc03a6abae)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 18f3d02fe4aabdd9579e626d0b0189e2e48bcb6a into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-de2c4e54ac7bc22d8bb1fbc2a16114d21ae4c4a9)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by darrynten** [comment]
This pull request **fixes 1 alert** when merging 18f3d02fe4aabdd9579e626d0b0189e2e48bcb6a into 3111999625df9c30aff08181090bfadebfae04e7 - [view on LGTM.com](https://lgtm.com/projects/g/darrynten/vue-quick-dash/rev/pr-de2c4e54ac7bc22d8bb1fbc2a16114d21ae4c4a9)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**